samples/SimpleWebSample/Program.cs

```diff
@@ -49,5 +42,15 @@ public static int Main(string[] args)
                 Log.CloseAndFlush();
             }
         }
+
+        public static IWebHost BuildWebHost(string[] args) =>
+            WebHost.CreateDefaultBuilder(args)
+                   .UseStartup<Startup>()
+                   .UseConfiguration(Configuration)
+                   .UseSerilog((context, configuration) =>
```

Comment:
This should just be the parameterless `UseSerilog()`, since the logger is already being created and configured inside `Main()`.

Reply:
Oh...excuse me. Fixed.